clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -64,7 +65,7 @@ def __getattr__(self, attr):
         global KUBERNETES_API_CACHE
         func = getattr(self._client, attr)
 
-        if attr in self.CACHED_FUNCTION_CALLS:
+        if os.environ.get('KUBE_CACHE_ENABLED', '') and attr in self.CACHED_FUNCTION_CALLS:
```

Comment:
I think this is roughly ok, but it feels kind of gross to me to be doing this somewhere kind of deep in the code.  How hard would it be to follow the pattern of, e.g., the `CMAN_CLUSTER` environment variable and do this as an arg?

Reply:
Discussed on slack -- using args would require passing a bunch of junk through, which feels ugly in its own way.